PusherPlatform.podspec

```diff
@@ -4,12 +4,13 @@ Pod::Spec.new do |s|
   s.summary          = 'Pusher Platform SDK in Swift'
   s.homepage         = 'https://github.com/pusher/pusher-platform-swift'
   s.license          = 'MIT'
-  s.author           = { "Hamilton Chapman" => "hamchapman@gmail.com" }
+  s.author           = { "Pusher Ltd." => "mobile-engineering@pusher.com" }
   s.source           = { git: "https://github.com/pusher/pusher-platform-swift.git", tag: s.version.to_s }
   s.social_media_url = 'https://twitter.com/pusher'
 
   s.requires_arc = true
   s.source_files = 'Sources/*.swift'
+  s.source_files = 'Platform/**/*.swift'
```

Comment:
Can you now get rid of `s.source_files = 'Sources/*.swift'`? (I think you renamed the `Sources` directory to `Platform`)

Reply:
Done.